src/components/ColumnSevenDayChange/index.js

```diff
@@ -0,0 +1,29 @@
+import { Arrow } from 'components/Arrow'
+import { Cell } from 'components/Cell'
+import { Column } from 'components/Column'

```

Comment:
import {Arrow, Cell, Column} from 'components'

Reply:
Refactored all folders / components / imports to follow this.